server/src/user/user.service.ts

```diff
@@ -26,4 +26,91 @@ export class UserService {
       throw error;
     }
   }
+
+  async getTodos(userId: number, houseId: number) {
+    try {
+      // Find the house of the user
+      const house = await this.prisma.house.findFirst({
+        where: { id: houseId },
+        select: { name: true },
+      });
```

Comment:
I think you also need to filter by userId to find a house!

Reply:
Oh yeah! Fixed it!
https://github.com/HAK2024/fairy-share/pull/61/commits/fa31417dfc85db04f33176c4bf7ea5906b511006